setup.py

```diff
@@ -17,11 +17,12 @@
 requirements = [
     "marshmallow>=3.10.0",
     "pymongo>=3.7.0",
+    "motor>=3.1.1",
```

Comment:
This shouldn't be a requirement. Let's keep it as an extra below.

Reply:
Done